pylivetrader/backend/alpaca.py

```diff
@@ -495,48 +499,49 @@ def get_bars(self, assets, data_frequency, bar_count=500):
             symbols = [assets.symbol]
         else:
             symbols = [asset.symbol for asset in assets]
+        # the timezone of end_dt if UTC, when data_frequency=daily, end_dt is midnight in UTC time
+        if not end_dt:
+            end_dt = pd.to_datetime('now', utc=True).floor('min')
```

Comment:
this entire block should be a function and not a block inside another function.
create a function with name `_get_current_end_dt` and write doc string inside explaining what you are doing and why